hexrd/ui/cal_tree_view.py

```diff
@@ -129,7 +129,10 @@ def recursive_add_tree_items(self, cur_config, cur_tree_item):
 
         for key in keys:
             if key == 'value':
-                self.set_value(key, cur_config[key], cur_tree_item)
+                data = cur_config[key]
+                if cur_tree_item.data(0) == 'tilt':
+                    data = [np.degrees(rad).item() for rad in cur_config[key]]
+                self.set_value(key, data, cur_tree_item)
```

Comment:
The underlying instrument config will likely be expecting the angles to be in radians.

If the user modifies the tilt values when they are degrees, will this convert them back to radians?

Reply:
Also note that if the Euler angle convention is "None", then we shouldn't convert to degrees.

Reply:
Thank you for catching that, the values were not being converted back to radians. Fixed conversion on value modification and only convert when the angle convention is `None`.